src/controller/subtitle-stream-controller.ts

```diff
@@ -190,11 +194,17 @@ export class SubtitleStreamController
       // Clear internal buffered lists
       this.tracksBuffered.forEach((_, index, tracks) => tracks[index] = []);
 
-      if (track.mode === 'showing') { 
-        track.mode = 'hidden';
-        setTimeout(() => {
-            track.mode = 'showing';
-        }, 10);
+      // This is for Xbox only to address persistent cue issue for Pluto.
+      // Without it, last timed text cue before the seek is called remains 
+      // on the screen even after seek is completed. See ticket below: 
+      // https://github.com/cbsinteractive/avia-player-support/issues/258
+      if (/Xbox; Xbox One/.test(navigator.userAgent)) {
```

Comment:
Really, this entire handler only applies to Xbox. Should we move this check to the top of the function and simple bail if its not Xbox? 

```js
if (!/Xbox; Xbox One/.test(navigator.userAgent)) {
  return;
}
```

Reply:
Good idea.. Let me isolate the whole thing, and not just the toggling of visibility block